docs/pages/index.mdx

```diff
@@ -102,10 +102,18 @@ Get started with Teleport Identity:
 ### Teleport Policy
 
 **Teleport Policy** unifies and controls access policies across all your
-infrastructure. With Teleport Policy’s Access Graph feature, you gain insights into role-based
-access control policies within Teleport and your cloud provider.
-
-Get started with [Teleport Policy](admin-guides/teleport-policy/teleport-policy.mdx).
+infrastructure. With Teleport Policy’s Access Graph feature, you gain insights
+into role-based access control policies within Teleport, your cloud providers,
+and critical infrastructure.
+
+- Get started with [Teleport
+  Policy](admin-guides/teleport-policy/teleport-policy.mdx).
+- Define [Crown Jewels](admin-guides/teleport-policy/crown-jewels.mdx) so you
+  can track changes to your most sensitive users and resources.
+- Enable [Teleport Policy
+  integrations](admin-guides/teleport-policy/integrations/integrations.mdx)
+  so you can import RBAC rules from AWS, Entra ID, and other sources and
+  visualize your entire RBAC configuration in on a single screen. 
```

Comment:
Good to add SSH key scanning as this is becoming a powerful feature.

Reply:
Added in 4b8ce13d26db069a0b050d53f65366dedbb498e3